### ISSUE TITLE
Can't Install 1.x branch releases

### ISSUE THREAD

**Issue by logancalldev** [Shopify/slate#402]
### Problem

Can't Install 1.x branch releases

### Replication steps

running `npm install -g @shopify/slate@v1.0.0-alpha.21`
or `npm install -g @shopify/slate@v1.0.0-alpha.23`

### Errors

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @shopify/slate@1.0.0-alpha.21
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/logan/.npm/_logs/2018-02-20T15_37_54_732Z-debug.log
```

Same output for alpha.23

*Any additional information which might be helpful.*

*npm 5.6*
*node v9.5.0*

**Comment by t-kelly** [comment]
@logancalldev please take a look at the Getting Started section of the [readme](https://github.com/Shopify/slate/tree/1.x)?

**Comment by logancalldev** [comment]
Thanks @t-kelly. The `yarn create` command threw me off, as I'd never seen it before. I was just looking for the `yarn add` or `npm install`. Thanks for your response.

**Comment by t-kelly** [comment]
Yeah it threw me off when I first saw it too -- but it's pretty cool, eh? You can also use the NPM equivalent, `npx`, which is included in NPM 5.2.0+
